src/main/java/application/Department.java

```diff
@@ -1,12 +1,41 @@
 package application;
 
+import java.util.Random;
+
 public class Department {
-	/*
-	 *
-	 * Your code comes here. Remember you can overload constructors.
-	 *
-	 */
+
+	String name;
+	int id;
+	String country;
+	String city;
+
+	public Department(String name, String country, String city) {
+		this.name= name;
+		this.id = getRandomID();
+		this.country = country;
+		this.city=city;
+	}
+
+	//default department
+	public Department() {
+		name = "Test";
+		id = getRandomID();
+		country = "Neverland";
+		city= "Gotham";
+	}
+
+
 	public boolean isEquals(Department other) {
-		return false;
+		if(id == other.id) {
+			return true;
+		} else {
+			return false;
+		}

```

Comment:
Never do this!!!!!

This can be one line

~~~java
return id == other.id
~~~